lua/nvim-treesitter/highlight.lua

```diff
@@ -48,6 +49,7 @@ hlmap["type"] = "Type"
 hlmap["type.builtin"] = "Type"
 hlmap["structure"] = "Structure"
 hlmap["include"] = "Include"
+hlmap["variable"] = "Identifier"
```

Comment:
i don't really agree with this, as "variable" will be the default highlighting for everything then.
What is your opinion on this @kyazdani42 ?

Reply:
well `Identifier` is defined as `any variable name` in vim documentation. not sure what you mean by 'the default highlighting

Reply:
Now, everything is red with my colorscheme. "Everything" should by rather neutral for my taste, so I would need to make `variable` to no hihglight after this change. How does this @vigoux @kyazdani42 look on your computers? If this change would make the highlighting unusable for most colorschemes we should not do it.

Reply:
basically identifier should not be used for variables as red is definitively not a good color. Identifier applies well on vimscript, but not on most languages, specially not python